web/protojson/options.go

```diff
@@ -0,0 +1,36 @@
+package protojson
+
+import "google.golang.org/protobuf/encoding/protojson"
+
+// MarshalingOptions has options for how web http middleware marshals data to the clients.
+type MarshalingOptions struct {
+	// Enable protojson encoding in API responses and any json encoding.
+	EnableProtoJSON bool
+
+	// UseProtoNames uses proto field name instead of lowerCamelCase name in JSON
```

Comment:
does it make sense to just embed or nest protojson.MarshalOptions here?

Reply:
Yeah changed the structure a bit. A bit simpler now